openghg/retrieve/icos/_retrieve.py

```diff
@@ -49,6 +50,8 @@ def retrieve_atmospheric(
                 - "never" - don't update mismatches and raise an AttrMismatchError
                 - "from_source" / "attributes" - update mismatches based on attributes from ICOS Header
                 - "from_definition" / "metadata" - update mismatches based on input metadata
+        force: If True, disregard previously retrieved hashes. Use this if you are unable to re-add
```

Comment:
In this circumstance this comment is true but the `force` key can also be used to replace "identical" data in general.

On the `new_object_store` branch this key word is defined as

> force: Force adding of data even if this is identical to data stored.

and I think that does fit with this definition as well in this case or could augment this to:

> force: Force adding of data even if this is identical to data stored (checked based on previously retrieved file hashes).